plugins/org.eclipse.fordiac.ide.export.ui/src/org/eclipse/fordiac/ide/export/properties/TypeExportPropertyPage.java

```diff
@@ -0,0 +1,250 @@
+/*******************************************************************************
+ * Copyright (c) 2025 Primetals Technologies Austria GmbH
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *   Mario Kastner
+ *     - initial API and implementation and/or initial documentation
+ *******************************************************************************/
+
+package org.eclipse.fordiac.ide.export.properties;
```

Comment:
Please still move this to `org.eclipse.fordiac.ide.export.ui.properties`:
```suggestion
package org.eclipse.fordiac.ide.export.ui.properties;
```